index.html

```diff
@@ -46,8 +46,169 @@ <h2>I'm a Software Developer</h2>
+                </p>
+                <div class="btn">
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="see-btn">
+                    <button>See Projects</button>
+                </div>
+            </div>
+            <div class="project-section">
+                <h2>Profesional Art <br>
+                    Printing Data
+                </h2>
+                <p>
+                    A daily selection of privately  <br> 
+                    personalized reads; no accounts or <br> 
+                    sign-ups required. Has been the <br> 
+                    industry's standard.
+                </p>
+                <div class="btn">
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="see-btn">
+                    <button>See Projects</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about-me-section">
+        <div class="about-me">
+            <h1>About me</h3>
+            <p>
+                Hello I’m a software developer! I can help <br>
+                you build a product , feature or website <br>
+                Look through some of my work and <br> 
+                experience! If you like what you see and 
+                <br> have a project you need coded, don’t 
+                <br> hesitate to contact me.
+            </p>
+            <div class="resume-btn">
+                <button>Get my resume</button>
+            </div>
+        </div>
+        <hr>
+        <div class="about_footer">
+            <div class="profile">
+               <ul>
+                   <li class="list-style Language">Language
+                       <ul>
+                            <li>JavaScript</li>
+                            <li>Ruby</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                    </li>
+                    <li class="list-style framework">Framework
+                        <ul>
+                            <li>React.js</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSpec</li>
+                            <li>Capybara</li>
+                         </ul>
+                     </li>
+                     <li class="list-style skills">Skills
+                        <ul>
+                            <li>Database Management</li>
+                            <li>Version</li>
+                            <li>CLI</li>
+                            <li>API</li>
+                            <li>Web Development</li>
+                         </ul>
+                     </li>
+               </ul>
```

Comment:
Fantastic work! :rocket: :clap:. Great job on making all technology categories & the technology in each category inside an `li` tag.